### ISSUE TITLE
Latest release 1.35.0 can't support terraform 1.0+ release?

### ISSUE THREAD

**Issue by wahaha2001** [IBM-Cloud/terraform-provider-ibm#3302]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version

Terraform version : 
Terraform v1.0.3
on darwin_amd64

Terraform IBM Provider Version : 1.35.0

### Affected Resource(s)

* ibm_schematics_workspace

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please share a link to the ZIP file.
```

### Debug Output

```
Initializing the backend...

Initializing provider plugins...
- Finding ibm-cloud/ibm versions matching ">= 1.35.0"...
- Finding cloudposse/utils versions matching "~> 0.11.0"...
- Installing ibm-cloud/ibm v1.35.0...
- Installed ibm-cloud/ibm v1.35.0 (self-signed, key ID AAD3B791C49CC253)
- Using previously-installed cloudposse/utils v0.11.1

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/cli/plugins/signing.html

Terraform has made some changes to the provider dependency selections recorded
in the .terraform.lock.hcl file. Review those changes and commit them to your
version control system if they represent changes you intended to make.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
complete : terraform init -input=false
╷
│ Error: "template_type" ("terraform_v1.0.3") should match regexp ^terraform_v0\.(?:11|12|13)(?:\.\d+)?$
│
│   with ibm_schematics_workspace.schematics_workspace["qa-automation"],
│   on main.tf line 71, in resource "ibm_schematics_workspace" "schematics_workspace":
│   71:   template_type = lookup(each.value.vars.schematics, "template_type", "terraform_v1.0.3")
│
╵
```

### Expected Behavior

ibm cloud provider can support terraform version 1.0+

### Steps to Reproduce

1. `terraform init`

### References

https://github.com/IBM-Cloud/terraform-provider-ibm/blob/21d604abd854c46e3c61626fbe72f1c22962f07d/ibm/resource_ibm_schematics_workspace.go#L465

* #0000

**Comment by kavya498** [comment]
similar issue
https://github.com/IBM-Cloud/terraform-provider-ibm/issues/3048

**Comment by kavya498** [comment]
available in latest 1.36.0 release..
closing this issue.. 
Thanks
